projects/sonar/src/app/app-routing.module.ts

```diff
@@ -401,4 +378,33 @@ export class AppRoutingModule {
       posParams: { type: new UrlSegment(url[1].path, {}) }
     };
   }
+
+  /**
+   * Get the ordered list of aggregations.
+   *
+   * @returns An observable resolving the ordered list.
+   */
+  private _documentAggregationsOrder(): Observable<any> {
+    return of(null).pipe(
+      switchMap(() => {
+        const view = this._route.snapshot.children[0].params.view;
+
+        let params = new HttpParams();
+        if (view) {
+          params = params.set('view', view);
+        }
+        if (this._route.snapshot.children[0].queryParams.collection_view) {
+          params = params.set('collection', '1');
+        }
+
+        return this._httpClient.get(
+          `${this._apiService.getEndpointByType(
```

Comment:
If I understood well, the aggregation is given by a specific backend url. Why not adding this in the serializer instead?